src/engine/physics/physic.cpp

```diff
@@ -0,0 +1,14 @@
+#include "engine/physics/physic.h"
+
+void Physic::linkPos(std::pair<double, double>* pos){
+   this->pos = pos;
+}
+
+void Physic::update(){
+    if(is_falling){
+	gravity(pos, &(phxParam.mass));
+    }
+
+    
```

Comment:
some useless empty lines here? (but could be address in #14 :slightly_smiling_face: )

Reply:
Yeah I decided to leave all the formatting issue so far, would create a lot of commits for not much + it is going to be adressed with some issues so I do not think it is a problem.

Reply:
seems logic!